R-package/tests/testthat/test_lgb.convert.R

```diff
@@ -75,20 +75,20 @@ test_that("lgb.prepare2() should work correctly for a dataset with numeric, fact
     )
     testDT <- data.table::as.data.table(testDF)
     for (input_data in list(testDF, testDT)) {
-        converted_dataset <- lgb.prepare2(input_data)
+        converted_dataset <- lgb.convert(input_data)
         expect_identical(class(input_data), class(converted_dataset))
         expect_identical(class(converted_dataset[["character_col"]]), "integer")
         expect_identical(class(converted_dataset[["factor_col"]]), "integer")
         expect_identical(converted_dataset[["character_col"]], c(1L, 2L, 3L))
         expect_identical(converted_dataset[["factor_col"]], c(1L, 1L, 2L))
 
-        # today, lgb.prepare2() does  not convert numeric  columns
+        # today, lgb.convert() does  not convert numeric  columns
```

Comment:
one whitespace instead of two whitespaces

Reply:
these have been fixed by accepting previous suggestions in the browser